web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -182,3 +182,29 @@ form .description
     &:before
       +icon-size(15px)
       content: '\e617'
+
+// Additional styling for dropbutton.
```

Comment:
Mobile is virtually un-usable:
![bs_realdroid_mobile_google nexus 6p-7 0-1080x1920](https://user-images.githubusercontent.com/1923476/28863402-03235ff6-7769-11e7-98b3-baf79bde70b5.jpg)